docs/src/userguide/loading_iris_cubes.rst

```diff
@@ -206,241 +206,8 @@ a specific ``model_level_number``::
     level_10 = iris.Constraint(model_level_number=10)
     cubes = iris.load(filename, level_10)
 
-Constraints can be combined using ``&`` to represent a more restrictive
-constraint to ``load``::
-
-    filename = iris.sample_data_path('uk_hires.pp')
-    forecast_6 = iris.Constraint(forecast_period=6)
-    level_10 = iris.Constraint(model_level_number=10)
-    cubes = iris.load(filename, forecast_6 & level_10)
-
-.. note::
-
-    Whilst ``&`` is supported, the ``|`` that might reasonably be expected is
-    not. Explanation as to why is in the :class:`iris.Constraint` reference
-    documentation.
-
-    For an example of constraining to multiple ranges of the same coordinate to
-    generate one cube, see the :class:`iris.Constraint` reference documentation.
-
-    To generate multiple cubes, each constrained to a different range of the
-    same coordinate, use :py:func:`iris.load_cubes`.
-
-As well as being able to combine constraints using ``&``,
-the :class:`iris.Constraint` class can accept multiple arguments,
-and a list of values can be given to constrain a coordinate to one of
-a collection of values::
-
-    filename = iris.sample_data_path('uk_hires.pp')
-    level_10_or_16_fp_6 = iris.Constraint(model_level_number=[10, 16], forecast_period=6)
-    cubes = iris.load(filename, level_10_or_16_fp_6)
```

Comment:
Strict Loading does cover using multiple constraints to produce multiple cubes, which isn't a replacement for 
> To generate multiple cubes, each constrained to a different range of the same coordinate, use :py:func:`iris.load_cubes`. 

but I think it is good enough. 
The intention behind the User Guide is that it is to introduce a beginner to Iris,and they could complete it all in ~ a day . So I think we need to take care to keep it small and avoid the easy-to-fall-into-trap of chucking a bunch of examples in there as we don't have anywhere else to put them.
I think the note about "to generate multiple cubes of constraining to a different range of the same coordinate" is not necessary for the user's guide. Also, admittedly, I struggled to find a place in the user guide where it would fit.

As for the second part
>the :class:`iris.Constraint` class can accept multiple arguments

there is an example of this: https://github.com/SciTools/iris/blame/main/docs/src/userguide/subsetting_a_cube.rst#L68
and
>a list of values can be given to constrain a coordinate to one of
a collection of values

This is covered in the API docs: https://github.com/SciTools/iris/blob/8e8a63c42e1ae5933f1bf71d790167c66c1704a1/lib/iris/_constraints.py#L58

Reply:
Thanks, I'm glad you gave this a lot of thought and it wasn't just an oversight 🙂